wsmaster/codenvy-hosted-api-license/pom.xml

```diff
@@ -42,13 +46,19 @@
             <artifactId>guava</artifactId>
         </dependency>
         <dependency>
+            <groupId>com.google.gwt</groupId>
```

Comment:
**gwt-user** library linked with **codenvy-machine-hosted**, which, in turn, is used by _CodenvyLicenseManager_ to calculate actual number of machine nodes:
```
[INFO] +- com.codenvy.plugin:codenvy-machine-hosted:jar:5.0.0-M8-SNAPSHOT:compile
...
[INFO] |  +- org.eclipse.che.plugin:che-plugin-docker-client:jar:5.0.0-M8-SNAPSHOT:compile
[INFO] |  |  +- com.google.gwt:gwt-user:jar:2.7.0:compile
[INFO] |  |  +- org.bouncycastle:bcpkix-jdk15on:jar:1.51:compile
[INFO] |  |  +- org.bouncycastle:bcprov-jdk15on:jar:1.51:compile
[INFO] |  |  \- org.eclipse.che.core:che-core-commons-gwt:jar:5.0.0-M8-SNAPSHOT:compile
[INFO] |  |     +- com.google.gwt:gwt-elemental:jar:2.7.0:compile
[INFO] |  |     \- org.vectomatic:lib-gwt-svg:jar:0.5.10:compile
```
```
[INFO] Compiling 25 source files to /home/ndp/projects/codenvy/wsmaster/codenvy-hosted-api-license/target/classes
/home/ndp/projects/codenvy/wsmaster/codenvy-hosted-api-license/src/main/java/com/codenvy/api/license/server/CodenvyLicenseManager.java:28: error: package com.codenvy.swarm.client does not exist
import com.codenvy.swarm.client.SwarmDockerConnector;
                               ^
/home/ndp/projects/codenvy/wsmaster/codenvy-hosted-api-license/src/main/java/com/codenvy/api/license/server/CodenvyLicenseManager.java:76: error: cannot find symbol
    private final SwarmDockerConnector    dockerConnector;
                  ^
  symbol:   class SwarmDockerConnector
  location: class CodenvyLicenseManager
/home/ndp/projects/codenvy/wsmaster/codenvy-hosted-api-license/src/main/java/com/codenvy/api/license/server/CodenvyLicenseManager.java:87: error: cannot find symbol
                                 SwarmDockerConnector dockerConnector,
                                 ^
  symbol:   class SwarmDockerConnector
  location: class CodenvyLicenseManager
3 errors
```